scripts/pull_github_pr.sh

```diff
@@ -25,7 +25,10 @@ NL=$'\n'
 
 PR_NUM=$1
 # convert full repo URL to its project/repo part:
-PROJECT=`git config --get remote.origin.url | sed 's/git@github.com://;s#https://github.com/##;s/\.git$//;'`
+REMOTE_SLASH_BRANCH="$(git rev-parse --abbrev-ref --symbolic-full-name  @{upstream})"
```

Comment:
Nitpick: I would add quotes around the `'@{upstream}`, the braces syntax is too similar to shell expansion of multiple filenames (bash doesn't have a problem with the unquoted version, so it's fine because the shebang at the top forces bash, but when I tried it in zsh it made a mess).